### ISSUE TITLE
Applying/Using models - issues

### ISSUE THREAD

**Issue by omesser** [iterative/mlem.ai#213]
Found several issues in [applying / using models](https://mlem.ai/doc/get-started/applying) page that make the experience either broken or a bit bumpy
## Issues
- [ ] With current latest code in example-mlem-getting-started

Creating the `predict.py` script as it appears in the page and running it, outputs a user warning:
```shell
❯ python predict.py
/Users/oded/iterative/test_workspace/example-mlem-get-started/.venv/lib/python3.8/site-packages/sklearn/base.py:450: UserWarning: X does not have valid feature names, but RandomForestClassifier was fitted with feature names
  warnings.warn(
[[0.47 0.24 0.29]]
```

- [ ] prediction results are not consistent with actual prediction:
in doc  it's `[[0.3, 0.3, 0.4]]`

- [ ] missing quoting/escaping in the `mlem apply` command. `[` needs to be escaped/quoted in `zsh` (it's ok in bash from what I've seen) so `pandas[csv]` -> `"pandas[csv]"`

```bash
❯ mlem apply models/rf new_data.csv -i --it pandas[csv]
zsh: no matches found: pandas[csv]
```

- [ ] `mlem apply` (CLI) does not output the results - see https://github.com/iterative/mlem/pull/470. 
```bash
❯ mlem apply models/rf new_data.csv -i --it "pandas[csv]"
⏳️ Importing object from new_data.csv
⏳️ Loading model from models/rf.mlem
🍏 Applying `predict` method...
❯
```
And after addressing the above `mlem` issue, it's invoking `predict` and not `predict_proba` method, so will return `[0]` which is not consistent with the above code example (it's good to see symmetrical behavior between api and cli here)

## Note - trying to debug why this happened
The added lines by `-v` aren't helpful - showing internal stuff (and skipping the info level output WAT) but at no point showing any print/peek of the data loaded from file which I would expect as a user in verbose logging, trying to debug my wrong usage. the json somewhat helped because its output showed that the prediction results are empty.
So, bottom line, Adding the `-v` verbose flag, didn't seem to help me, a user, understand why I don't see my prediction result as I would've expected, or why my prediction results are different in CLI than in code
(see https://github.com/iterative/mlem/pull/470)
```bash
❯ mlem -v apply models/rf new_data.csv -i --it "pandas[csv]" --json
2022-11-06 18:32:07,540 [DEBUG] mlem.core.hooks: Registering NumpyNumberType to DataAnalyzer
2022-11-06 18:32:07,541 [DEBUG] mlem.core.hooks: Registering NumpyNdarrayType to DataAnalyzer
2022-11-06 18:32:07,542 [DEBUG] mlem.core.hooks: Not registerting _PandasDataType to any Analyzer because it's an abstract class
2022-11-06 18:32:07,543 [DEBUG] mlem.core.hooks: Registering SeriesType to DataAnalyzer
2022-11-06 18:32:07,544 [DEBUG] mlem.core.hooks: Registering DataFrameType to DataAnalyzer
2022-11-06 18:32:07,549 [DEBUG] mlem.core.hooks: Registering PandasImport to ImportAnalyzer
2022-11-06 18:32:07,811 [DEBUG] mlem.core.hooks: Registering SklearnModel to ModelAnalyzer
2022-11-06 18:32:07,812 [DEBUG] mlem.core.hooks: Registering SklearnPipelineType to ModelAnalyzer
[0]
```

**Comment by omesser** [comment]
We should probably move away from iris as mentioned here, but in the meantime can be averted by changing training to ditch the feature names for training:
```py
    rf.fit(data.values, y.values)

    save(
        rf,
        "models/rf",
        sample_data=data.values,
    )
```

**Comment by omesser** [comment]
First 3 addressed by #214
